packages/components/buttongroup/src/ButtonGroup.styles.ts

```diff
@@ -0,0 +1,71 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import type { ButtonGroupVariants } from './types';
+
+const getGroupContentStyle = (variant: ButtonGroupVariants) => {
+  switch (variant) {
+    case 'separate':
+      return {
+        marginLeft: tokens.spacingS,
+        marginRight: tokens.spacingS,
+        '&:first-child': {
+          marginLeft: 0,
+        },
+        '&:last-child': {
+          marginRight: 0,
+        },
+      };
+  }
+
+  return {
+    borderRadius: '0 !important',
+    marginRight: '-1px',
+    zIndex: tokens.zIndexDefault,
+    '&:first-child': {
+      borderBottomLeftRadius: `${tokens.borderRadiusMedium} !important`,
+      borderTopLeftRadius: `${tokens.borderRadiusMedium} !important`,
+    },
+    '&:last-child': {
+      borderBottomRightRadius: `${tokens.borderRadiusMedium} !important`,
+      borderTopRightRadius: `${tokens.borderRadiusMedium} !important`,
```

Comment:
Uhmmm do we need to use `!important` here because the way emotion handle CSS classes?

Is there something we could do to avoid them?

Reply:
This is because for some reason on gatsby the classes aren't being merged so the default border-radius of the button was overwriting this one.
I'm taking a deeper look to see if it's due to a plugin version or something, but on storybook they render correctly even without the `!important` flag

Reply:
Actually it's my bad, I was calling the `css()` method in the same value twice, fixed it and seems to be working fine on gatsby locally.